docs/user/get-started/ios.md

```diff
@@ -0,0 +1,105 @@
+# Adding Glean to your iOS project
+
+## Usage
+
+### Integrating with your project
+
+#### Setting up the dependency
+
+Glean can be consumed through [Carthage](https://github.com/Carthage/Carthage), a dependency manager for macOS and iOS.
+For consuming the latest version of Glean, add the following line to your `Cartfile`::
+
+```
+github "mozilla/glean" "master"
+```
+
+#### Integrating with the build system
+
+In order for the Glean SDK to generate an API for your metrics, a Python environment must be accessible at build time.
+
+**TODO: Fill in how to add it to the build system**
+
+### Adding new metrics
+
+All metrics that your project collects must be defined in a `metrics.yaml` file.
+Add this file to your project and define it as an input file for the `sdk_generator.sh` script in the `Run Script` step defined before.
+The format of that file is documented [with `glean_parser`](https://mozilla.github.io/glean_parser/metrics-yaml.html).
+To learn more, see [adding new metrics](adding-new-metrics.md).
+
+> **Important**: as stated [before](../adding-glean-to-your-project.md#before-using-glean), any new data collection requires documentation and data-review.
+> This is also required for any new metric automatically collected by the Glean SDK.
+
+### Adding custom pings
+
+Please refer to the [custom pings documentation](../pings/custom.md).
+
+> **Important**: as stated [before](../adding-glean-to-your-project.md#before-using-glean), any new data collection requires documentation and data-review.
+> This is also required for any new metric automatically collected by the Glean SDK.
+
+### Testing metrics
+
+In order to make testing metrics easier 'out of the box', all metrics include a set of test API functions in order to facilitate unit testing.  These include functions to test whether a value has been stored, and functions to retrieve the stored value for validation.  For more information, please refer to [Unit testing Glean metrics](../testing-metrics.md).
+
+### Adding metadata about your project to the pipeline
+
+In order for data to be collected from your project, metadata must be added to `probe_scraper`.
+
+These specific steps are described in [the `probe_scraper` documentation](https://github.com/mozilla/probe-scraper#adding-a-new-glean-repository).
+
+## Application-specific steps
+
+The following steps are required for applications using the Glean SDK, but not libraries.
+
+### Initializing the Glean SDK
+
+The Glean SDK should only be initialized from the main application, not individual libraries.
+If you are adding Glean support to a library, you can safely skip this section.
+Please also note that the Glean SDK does not support use across multiple processes, and must only be initialized on the application's main process.
+
+Before any data collection can take place, the Glean SDK **must** be initialized from the application.
```

Comment:
Maybe out of scope for this but is this _really_ true?  I know we would like them to initialize Glean as early as possible but we can still queue data collection tasks before init.

Reply:
Good catch! This is indeed not true anymore, thanks to your work.